potodo/_github.py

```diff
@@ -1,32 +1,48 @@
+import re
 import requests
 
-from subprocess import getoutput
+from typing import Mapping
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """Tries to get the repository url from git commands
+    """
+    url: str = check_output(
```

Comment:
Is it really needed by mypy to type this one?

Reply:
I expect typecheckers to infer the type from check_output typing anotation if any.

Reply:
🤷‍♂ idk, is it bad that it's here ? `check_output` is defined as returning a `str` anyway so idk

Reply:
Should I remove it then ?

Reply:
I'd say: if it's not needed don't put it.